c10/util/typeid.h

```diff
@@ -520,6 +520,7 @@ class C10_API TypeMeta final {
 
   static uint16_t existingMetaDataIndexForType(TypeIdentifier identifier);
 
+ public:
```

Comment:
I debated adding _DO_NOT_CALL_DIRECTLY to the name of addTypeMetaData, but that didn't seem very PyTorchy.

Reply:
meh, I don't mind haha